tempodb/encoding/vparquet4/wal_block_test.go

```diff
@@ -224,7 +224,7 @@ func TestWalBlockFindTraceByID(t *testing.T) {
 			require.NoError(t, err)
 			require.NotNil(t, found)
 			require.True(t, proto.Equal(trs[i], found.Trace))
-			require.Greater(t, found.Metrics.InspectedBytes, uint64(40000)) // approximate value
```

Comment:
can you make it something like 100, or a low value so we test that InspectedBytes has a value?